common/djangoapps/third_party_auth/migrations/0001_squashed_0026_auto_20200401_1932.py

```diff
@@ -0,0 +1,363 @@
+                ('name', models.CharField(help_text='Name of this provider (shown to users)', max_length=50)),
+                ('secondary', models.BooleanField(default=False, help_text='Secondary providers are displayed less prominently, in a separate list of "Institution" login providers.')),
+                ('skip_registration_form', models.BooleanField(default=False, help_text='If this option is enabled, users will not be asked to confirm their details (name, email, etc.) during the registration process. Only select this option for trusted providers that are known to provide accurate user information.')),
+                ('skip_email_verification', models.BooleanField(default=False, help_text='If this option is selected, users will not be required to confirm their email, and their account will be activated immediately upon registration.')),
+                ('backend_name', models.CharField(db_index=True, help_text='Which python-social-auth OAuth2 provider backend to use. The list of backend choices is determined by the THIRD_PARTY_AUTH_BACKENDS setting.', max_length=50)),
+                ('key', models.TextField(blank=True, verbose_name='Client ID')),
+                ('secret', models.TextField(blank=True, help_text='For increased security, you can avoid storing this in your database by leaving  this field blank and setting SOCIAL_AUTH_OAUTH_SECRETS = {"(backend name)": "secret", ...} in your instance\'s Django settings (or lms.auth.json)', verbose_name='Client Secret')),
+                ('other_settings', models.TextField(blank=True, help_text='Optional JSON object with advanced settings, if any.')),
+                ('provider_slug', models.SlugField(default='temp', help_text='A short string uniquely identifying this provider. Cannot contain spaces and should be a usable as a CSS class. Examples: "ubc", "mit-staging"', max_length=30)),
+                ('changed_by', models.ForeignKey(editable=False, null=True, on_delete=django.db.models.deletion.PROTECT, to=settings.AUTH_USER_MODEL, verbose_name='Changed by')),
+                ('icon_image', models.FileField(blank=True, help_text='If there is no Font Awesome icon available for this provider, upload a custom image. SVG images are recommended as they can scale to any size.', upload_to='')),
+                ('visible', models.BooleanField(default=False, help_text='If this option is not selected, users will not be presented with the provider as an option to authenticate with on the login screen, but manual authentication using the correct link is still possible.')),
+            ],
+            options={
+                'verbose_name': 'Provider Configuration (OAuth)',
+                'verbose_name_plural': 'Provider Configuration (OAuth)',
+            },
+        ),
+        migrations.RunPython(
+            code=fill_oauth2_slug,
+            reverse_code=django.db.migrations.operations.special.RunPython.noop,
+        ),
+        migrations.AddField(
```

Comment:
There are a ton of AddFields.  I wonder if that is because of the RunPython?  I wonder if there is a way to temporarily get rid of that to get a fresh initial or fresh squash?